runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -184,6 +191,7 @@ public CorfuCompileProxy(CorfuRuntime rt, UUID streamID, Class<T> type, Object[]
                         .nextToken(Collections.singleton(streamID), 0).getToken()
                         .getTokenValue();
         log.debug("Access[{}] conflictObj={} version={}", this, conflictObject, timestamp);
+        correctness.info("Version, {}", timestamp);
```

Comment:
Tracing is off by default. It would generate way too much logging (especially for longevity run). Turning on is only for debugging session when an issue has been detected.

Also, having them outputted to a specific file simplifies the recollection + off-line verification.